integration/src/main/resources/servers/catalog-local.json

```diff
@@ -3,6 +3,7 @@
   "description": "Catalog Local env.",
 
   "catalogUri": "http://localhost:8080",
+  "datarepoUri": "https://jade.datarepo-dev.broadinstitute.org/",
```

Comment:
Do we need this to change depending on which environment we're running in?

Reply:
`local` works for both local development and `dev` tests, since the dev run is "local" to the Github runner it runs on.

I think we do need additional server configs for the various test environments we need to support. I can make another ticket for that work, and the work to find out how to have QA run them.

Reply:
Created https://broadworkbench.atlassian.net/browse/DC-299